clientgui/DlgAdvPreferences.cpp

```diff
@@ -1011,6 +1011,21 @@ bool CDlgAdvPreferences::IsValidTimeValue(const wxString& value) {
             return false;
         }
     }
+    //verify correct diapasone
```

Comment:
This is hard to translate for non English speakers. I had to look it up in the Oxford dictionary and found out it means "range, or scope". A more simpler sentence would be better here.

"verify correct format and range of time values"

Reply:
Fixed